contracts/WethToBscStrategy.sol

```diff
@@ -26,6 +26,8 @@ contract WethToBscStrategy is BaseStrategy {
     address public ethDepositToBsc =
         address(0x13B432914A996b0A48695dF9B2d701edA45FF264);
 
+    uint256 internal _balanceOfWantOnBSC = 0;
```

Comment:
underscore is used of read-only, since you are modifying, plz remove the `_`.

Reply:
Is there a good style guide somewhere?